packages/redeem_solver/src/ika_client.ts

```diff
@@ -0,0 +1,176 @@
+import {
+	IkaClient as SdkIkaClient,
+	IkaTransaction,
+	getNetworkConfig,
+	Curve,
+	SignatureAlgorithm,
+	Hash,
+	SessionsManagerModule,
+	CoordinatorInnerModule,
+	createUserSignMessageWithCentralizedOutput,
+	type IkaConfig,
+} from "@ika.xyz/sdk";
+import type { SuiNet } from "@gonative-cc/lib/nsui";
+import type { SuiClient as MystenClient } from "@mysten/sui/client";
+import {
+	Transaction,
+	type TransactionArgument,
+	type TransactionObjectArgument,
+} from "@mysten/sui/transactions";
+
+export interface IkaClient {
+	initialize(): Promise<void>;
```

Comment:
instead of exposing this method via interface, we should have an async function (could be a static function or global scope function) that constructs the object